package.json

```diff
@@ -37,8 +38,7 @@
     "start": "yarn storybook",
     "storybook": "node verify-node-version.js && concurrently --raw --kill-others 'yarn relay --watch' 'start-storybook -p 9001'",
     "sync-colors": "cd externals/elan && git fetch && git checkout origin/master && cp components/lib/variables/colors.json ../../data",
-    "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && cp .env.example .env && yarn install && yarn dump-schema -- ../../data",
-    "sync-schema:localhost": "cd ../metaphysics && yarn dump-schema -- ../reaction/data",
+    "sync-schema": "graphql-fetch-schema https://metaphysics-staging.artsy.net -o data",
```

Comment:
`graphql-cli get-schema` seems to be a bit broken right now (willing to debug if its a blocker or can factor it back in when we need something more extensive since this does exactly what it says.)